File: C++/12.cpp
#include <iostream>
#include <cmath>

using namespace std;

double AreaOfTriangle(double a, double b, double c)
{
    double s = (a + b + c) / 2;

    double area = sqrt(s * (s - a) * (s - b) * (s - c));
    return area;
}

int main()
{
    double a, b, c;
    cout << "Enter the lengths of the three sides of the triangle: ";
    cin >> a >> b >> c;

    if (AreaOfTriangle(a, b, c) == 0)
        cout << "The triangle doesnt exist";

    else
        cout << "The area of the triangle: " << AreaOfTriangle(a, b, c) << endl;

}

File: Python/12.py
import math as m


def AreaOfTriangle(a, b, c):
    s = (a + b + c) / 2
    area = m.sqrt(s * (s - a) * (s - b) * (s - c))
    return area


print("Enter the sides of the triangle: ")
a = int(input())
b = int(input())
c = int(input())

if (AreaOfTriangle(a, b, c) == 0):
    print("The triangle doesnt exist")
else:
    print("The area of the triangle is:", AreaOfTriangle(a, b, c))
